src/main/java/core/basesyntax/service/FileReaderService.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service;
+
+import core.basesyntax.exception.FruitException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderService {
```

Comment:
Let's create an interface for this and the following services. Implementations should be placed in 'impl' package, as described in Common Mistakes.